### PR TITLE
[SPARK-37703][BUILD] Upgrade SBT to 1.5.8

### PR DESCRIPTION
**Pull request by dchvn**
### What changes were proposed in this pull request?
This PR aims to upgrade SBT to 1.5.8

### Why are the changes needed?
 SBT 1.5.8 was released, which includes a fix for CVE-2021-45105.

### Does this PR introduce _any_ user-facing change?
No

### How was this patch tested?
Pass the CIs.

### CODE REVIEW

File: project/build.properties [hunk 1]
```diff
@@ -15,4 +15,4 @@
 # limitations under the License.
 #
 # Please update the version in appveyor-install-dependencies.ps1 together.
```
Comment (HyukjinKwon) [inline project/build.properties:17]: Please update the version in appveyor-install-dependencies.ps1 together.

**Review by HyukjinKwon — APPROVED** [review]

**Review by dongjoon-hyun — APPROVED** [review]
+1, LGTM. Thank you, @dchvn and @HyukjinKwon .
Merged to master.

**Comment by dongjoon-hyun** [comment]
Ur, it seems that this company still ban the personal credits by using `dgd_contributor@viettel.com.vn`.

```
 Authored-by: dch nguyen <dgd_contributor@viettel.com.vn>
```

Maybe, I lost the context about the previous discussion. @HyukjinKwon . Is this the final conclusion with this company?

**Comment by HyukjinKwon** [comment]
Hm, this is slightly odd actually. The final decision was that to add `dgd_contributor@viettel.com.vn`, _as a secondary email_ to @dchvn's personal account, and use his own email as a primary email address. Seems like `dgd_contributor@viettel.com.vn` is set as a primary account.

@dchvn mind changing the primary email address in your account as your personal email address? Please change it here at https://github.com/settings/profile
Screen Shot 2021-12-22 at 9 09 23 AM

Also, I would encourage to not to use a company image in the profile picture since this is your own personal account, and it would be great to make sure this account is used individually as we discussed.

**Comment by dongjoon-hyun** [comment]
Thank you for the summary, @HyukjinKwon !

**Comment by dchvn** [comment]
I updated as your suggestion. Thank you, @dongjoon-hyun @HyukjinKwon !

**Pull request closed without merging**